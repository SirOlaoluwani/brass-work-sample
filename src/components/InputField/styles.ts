import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
  container: {
    height: 50,
    backgroundColor: "white",
    marginBottom: 16,
    borderRadius: 10,
    borderWidth: 1,
    borderColor: "#9e9e9e47",
    padding: 10,
  },
});
