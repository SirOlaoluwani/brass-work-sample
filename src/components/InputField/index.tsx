import React from "react";
import { TextInput, TextInputProps } from "react-native";
import { styles } from "./styles";

const InputField = (props: TextInputProps) => {
  return <TextInput style={styles.container} {...props} />;
};

export default InputField;
