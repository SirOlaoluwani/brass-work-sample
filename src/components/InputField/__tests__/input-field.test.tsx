import React from "react";
import renderer from "react-test-renderer";

import InputField from "..";

describe("<InputField />", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<InputField />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
