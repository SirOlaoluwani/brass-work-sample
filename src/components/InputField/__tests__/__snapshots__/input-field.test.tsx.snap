// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputField /> matches the snapshot 1`] = `
<TextInput
  allowFontScaling={true}
  rejectResponderTermination={true}
  style={
    Object {
      "backgroundColor": "white",
      "borderColor": "#9e9e9e47",
      "borderRadius": 10,
      "borderWidth": 1,
      "height": 50,
      "marginBottom": 16,
      "padding": 10,
    }
  }
  underlineColorAndroid="transparent"
/>
`;
